src/components/NoClassMessage.vue

```diff
@@ -24,5 +26,19 @@ export default {
       AFLogoSvg: assets.AFLogoSvg,
     };
   },
+  computed: {
+    displayMessage() {
+      return this.isLanguageSelectedEnglish
+        ? "There is no class right now."
```

Comment:
we wanted to change this to `There is no session right now` na

Reply:
Separate PR